src/pages/restaurant/Restaurant.js

```diff
@@ -93,7 +75,8 @@ class Restaurant extends React.Component {
     return <Menu dishes={dishes} />;
```

Comment:
try to access store from respective component instead of sending as props
but if its not deep and props are not drilled then its fine